homework5MatrixMethod/GetWordsCountInSentence.java

```diff
@@ -0,0 +1,22 @@
+package homework5MatrixMethod;
+
+public class GetWordsCountInSentence {
+    public static void main(String[] args) {
+        String stringToCheck = "sentence to test code of words count";
+
+        boolean isInWordCheckPhase = true;
+        int wordsCount = 0;
+
+        for (int i = 0; i < stringToCheck.length(); i++) {
+            char charToCheck = stringToCheck.charAt(i);
+
+            if ((97 <= charToCheck && charToCheck <= 122 || 65 <= charToCheck && charToCheck <= 90) && !isInWordCheckPhase) {
```

Comment:
change this numbers with chars

Reply:
String stringToCheck = "sentence  test";
Words cont in sentence: 1
check the logic again